themes/bootstrap5/js/embedded_record.js

```diff
@@ -42,168 +42,115 @@ VuFind.register('embedded', function embedded() {
     }
   }
 
-  /**
-   * Load the content for a specific record tab via AJAX.
-   * @param {string}  tabid    The ID of the tab to load.
-   * @param {boolean} [_click] Whether to trigger a click on the tab after loading (default = false).
-   * @returns {boolean} Returns false if the tab redirects to a new page, otherwise true.
-   */
-  function ajaxLoadTab(tabid, _click) {
-    var click = _click || false;
-    var $tab = $('#' + tabid);
-    var $result = $tab.closest('.result');
-    if ($result.length === 0) {
-      return true;
-    }
-    var id = $result.find('.hiddenId')[0].value;
-    var source = $result.find('.hiddenSource')[0].value;
-    if ($tab.parent().hasClass('noajax')) {
-      window.location.href = $tab.is('a')
-        ? $tab.attr('href') // tab case
-        : $tab.find('a').attr('data-href'); // accordion case
-      return false;
-    }
-    var urlroot;
-    if (source === VuFind.defaultSearchBackend) {
-      urlroot = 'Record';
-    } else {
-      urlroot = source.charAt(0).toUpperCase() + source.slice(1).toLowerCase() + 'Record';
-    }
-    if (!$tab.hasClass('loaded')) {
-      $('#' + tabid + '-content').html(VuFind.loading());
-      var tab = tabid.split('_');
-      tab = tab[0];
-      $.ajax({
-        url: VuFind.path + '/' + urlroot + '/' + encodeURIComponent(id) + '/AjaxTab',
-        type: 'POST',
-        data: { tab: tab },
-        success: function ajaxTabSuccess(data) {
-          var html = data.trim();
-          if (html.length > 0) {
-            $('#' + tabid + '-content').html(VuFind.updateCspNonce(html));
-            VuFind.emit('record-tab-init', {container: document.querySelector('#' + tabid + '-content')});
-          } else {
-            $('#' + tabid + '-content').html(VuFind.translate('collection_empty'));
-          }
-          $('#' + tabid).addClass('loaded');
-        }
-      });
-    }
-    if (click && !$tab.parent().hasClass('default')) {
-      $tab.trigger("click");
-    }
-    return true;
-  }
-
   /**
    * Toggle the embedded detailed view of a record.
-   * @param {jQuery} _link   The link element.
-   * @param {string} [tabid] The ID of the tab to open (default = first available tab).
+   * @param {HTMLElement} link   The link element.
+   * @param {string} [tabId] The ID of the tab to open (default = first available tab).
```

Comment:
Should these comments align?

```suggestion
   * @param {HTMLElement} link The link element.
   * @param {string} [tabId]   The ID of the tab to open (default = first available tab).
```

Reply:
In JS we do not align the parameter doc comments. But I removed the extra whitespace.